src/main/java/uk/gov/companieshouse/confirmationstatementapi/checks/CompanyPscCheck.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.companieshouse.confirmationstatementapi.checks;
```

Comment:
this might be better placed in the eligibility package implementing the eligibility rule interface.
That way we can just add it to the list of rules that a company needs to pass tp be eligible.

Reply:
I had initially, but realised it is a navigation control and not really an eligibility control. So i moved it out coz it didn't fit the EligibilityRule signature (never returns a 400 status).

Reply:
This would be part of the eligibility checks, the spec currently doesn't contain what the response would be.
But if the company has too many PSC's, then we'd want to return 400 with a body something along the lines of "INVALID_COMPANY_TOO_MANY_PSCS"

Reply:
The eligibility rules are added in the order you want them to execute [here](https://github.com/companieshouse/confirmation-statement-api/blob/main/src/main/java/uk/gov/companieshouse/confirmationstatementapi/configuration/ConfirmationStatementServiceEligibilityConfig.java#L43)

And they are then ran sequentially [here](https://github.com/companieshouse/confirmation-statement-api/blob/main/src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/ConfirmationStatementService.java#L40)

So adding this as an eligibility rule that is later in the list than company status and company type rules would achieve the required effect.